mkdocs/serve.py

```diff
@@ -71,7 +71,7 @@ def builder():
             theme=theme,
         )
         config['site_dir'] = tempdir
-        build(config, live_server=True)
+        build(config, live_server=True, clean_site_dir=True)
```

Comment:
Would it perhaps make more sense to remove/delete the temp dir when the server is shut down? Then the tmp dir wouldn't be hanging around littering the file system when not in use and the next time the `serve` command is used it will always be "clean."